### PR TITLE
Track aggregation script time

### PR DESCRIPTION
**Pull request by sravfeyn**
@emord version of https://github.com/dimagi/commcare-hq/pull/20156

cc @orangejenny

### CODE REVIEW

**Comment by emord** [comment]
Looks good but there's a conflict with merges

**Comment by sravfeyn** [comment]
@emord should be good now

**Comment by emord** [comment]
```
======================================================================
ERROR: test suite for <corehq.tests.nose.HqdbContext object at 0x7f668ca68c10>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 296, in setUp
    self.setupContext(context)
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/vendor/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/mnt/commcare-hq-ro/corehq/tests/nose.py", line 214, in setup
    super(HqdbContext, self).setup()
  File "/usr/local/lib/python2.7/site-packages/django_nose/plugin.py", line 423, in setup
    self.old_names = self.runner.setup_databases()
  File "/vendor/lib/python2.7/site-packages/django/test/runner.py", line 546, in setup_databases
    self.parallel, **kwargs
  File "/vendor/lib/python2.7/site-packages/django/test/utils.py", line 187, in setup_databases
    serialize=connection.settings_dict.get('TEST', {}).get('SERIALIZE', True),
  File "/vendor/lib/python2.7/site-packages/django/db/backends/base/creation.py", line 69, in create_test_db
    run_syncdb=True,
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 131, in call_command
    return command.execute(*args, **defaults)
  File "/vendor/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 99, in handle
    "'python manage.py makemigrations --merge'" % name_str
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0045_add_new_wasting_stunting_to_chm, 0042_aggregatesqlprofile in icds_reports).
To fix them run 'python manage.py makemigrations --merge'
----------------------------------------------------------------------
```

**Comment by sravfeyn** [comment]
Reordered the migrations now, should be good after build pass.

**Comment by sravfeyn** [comment]
@emord

File: custom/icds_reports/utils/__init__.py [hunk 1]
```diff
@@ -544,3 +546,21 @@ def person_has_aadhaar_column(beta):
...
+    from custom.icds_reports.models import AggregateSQLProfile
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        start = time.time()
+        result = func(*args, **kwargs)
+        end = time.time()
+        AggregateSQLProfile(
+            name=str(func),
+            duration=int(end - start)
```
Comment (emord) [inline custom/icds_reports/utils/__init__.py]: postgres/django has a date time range field that might have been good for this https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#datetimerangefield

**Pull request merged**